src/index.ts

```diff
@@ -24,6 +24,7 @@ const commandModules = [
   "hld",
   "infra",
   "project",
+  "ring",
```

Comment:
LGTM! Only question is if we should expose the rings commands right now.

Reply:
comment out this liner until the commands are ready :-)
uncomment it during development. and unit tests will continue will run regardless :-)